core/common/src/ConcurrentQuery.ts

```diff
@@ -24,7 +24,7 @@ export enum QueryRowFormat {
    */
   UseECSqlPropertyIndexes,
   /** Each row is an object in which each non-null column value can be accessed by a [remapped property name]($docs/learning/ECSqlRowFormat.md).
-   * This format is backwards-compatible with the format produced by iModel.js version 2.x. Null values are omitted.
+   * This format is backwards-compatible with the format produced by version 2.x. Null values are omitted.
```

Comment:
Reader may not be clear what "version" refers to - be specific?

Reply:
Fix to `by iTwin.js 2.x.`. Let me know what you think.